README.md

```diff
@@ -697,16 +697,17 @@ Any incoming request that has a request body that exceeds this limit will be
 rejected with a `413` (Request Entity Too Large) error message without calling
 the next middleware handlers.
 
-Any incoming request that does not have its size defined and uses the (rare)
-`Transfer-Encoding: chunked` will be rejected with a `411` (Length Required)
-error message without calling the next middleware handlers.
-Note that this only affects incoming requests, the much more common chunked
-transfer encoding for outgoing responses is not affected.
-It is recommended to define a `Content-Length` header instead.
-Note that this does not affect normal requests without a request body
-(such as a simple `GET` request).
+The `RequestBodyBufferMiddleware` will buffer requests with bodies of known size 
+(i.e. with `Content-Length` header specified) as well as requests with bodies of 
+unknown size (i.e. with `Transfer-Encoding: chunked` header). To simplify request 
+handling for downstream middlewares or applications buffered `chunked` requests will 
+have the `Transfer-Encoding` header removed and a `Content-Length` header with the 
+actual request body size received added instead.
+
+Note that this only affects incoming requests, the chunked transfer encoding for 
+outgoing responses is not affected.
```

Comment:
This sentence made much more sense for the old behavior where these requests were rejected. Does it make sense to you to remove this sentence?

Reply:
Yep, removed.